app/kumactl/cmd/install/render_helm_files.go

```diff
@@ -17,6 +17,28 @@ import (
 	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
 )
 
+type fileFilter func(data.File) bool
+
+var helmTemplateFilters []fileFilter
+
+func filterHelmTemplates(files data.FileList) data.FileList {
+	var filteredFiles data.FileList
+	for _, template := range files {
+		var filtered bool
+		for _, f := range helmTemplateFilters {
+			filtered = filtered || !f(template)
```

Comment:
"filtered" the local bool is used to mean "filter out this element", but the filters in "helmTemplateFilters" are used to mean "filter in this element". So we should either use different words or align the meanings.